transformers/struct.go

```diff
@@ -348,6 +350,9 @@ func defaultGoTypeToSchemaType(v reflect.Type) (arrow.DataType, error) {
 		}
 		return arrow.ListOf(elemValueType), nil
 
+	case reflect.Interface:
+		return nil, nil // silently ignore
```

Comment:
This ensures we don't panic now that we don't ignore this kind in `isTypeIgnored`.